tests/runtests.sh

```diff
@@ -180,6 +181,18 @@ if [ "" != "$SELENIUM_TIMEOUT" ]; then
     timeout_arg="--timeout=$SELENIUM_TIMEOUT"
 fi
 
+if [ "" != "$POKYDIR" ]; then
+    local_arg="--local"
+    poky_bind="-v $POKYDIR:/workdir/poky"
+fi
+
+    [[ -d $tempdir ]] || tempdir=$(mktemp -d --suffix toastertest --tmpdir)
```

Comment:
Whoops leftover from earlier version where I let it be set. Will fix in
morning.

On Aug 25, 2016 8:21 PM, "Randy Witt" <notifications@github.com> wrote:

> In tests/runtests.sh
> <https://github.com/crops/toaster-container/pull/9#discussion_r76359376>:
>
> > @@ -180,6 +181,18 @@ if [ "" != "$SELENIUM_TIMEOUT" ]; then
> >      timeout_arg="--timeout=$SELENIUM_TIMEOUT"
> >  fi
> >
> > +if [ "" != "$POKYDIR" ]; then
> > +    local_arg="--local"
> > +    poky_bind="-v $POKYDIR:/workdir/poky"
> > +fi
> > +
> > +    [[ -d $tempdir ]] || tempdir=$(mktemp -d --suffix toastertest --tmpdir)
>
> How would the "-d" ever be true? I don't see tempdir ever being set
> anywhere else.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/crops/toaster-container/pull/9/files/9dda39d3cf58b81c6820ef1528d2728a962f26a1..f0969b0f75b67f6c545550ec3a9138363e74d671#r76359376>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACkxjfCGs5esmQfMYB8JamLV5I4jGGKNks5qjlu8gaJpZM4JtQ5b>
> .
>